src/main/java/org/ballerinax/datamapper/DataMapperPlugin.java

```diff
@@ -90,16 +97,17 @@ public void init(DiagnosticLog diagnosticLog) {
 
     @Override
     public void process(PackageNode packageNode) {
-        Iterator<TopLevelNode> iterator = ((BLangPackage) packageNode).topLevelNodes.stream().filter(topLevelNodes -> topLevelNodes instanceof BLangTypeDefinition).iterator();
+        Iterator<TopLevelNode> iterator = ((BLangPackage) packageNode).topLevelNodes.stream().filter(topLevelNodes ->
```

Comment:
Shall we replace this stream with a plain java for(TopLevelNode node : ((BLangPackage) packageNode).topLevelNodes) {
// code
}

We usually try to avoid java streams due to performance reasons.

Reply:
Ok, Done